app/containers/UserPage/saga.js

```diff
@@ -10,7 +10,7 @@ export function* getUser(action) {
   if (response.id !== undefined) {
     yield put(getUserSuccess(response));
   } else {
-    yield put(getUserFailure());
+    yield put(signOut());
```

Comment:
Not in every situation you have to log out the user. Maybe the request returns 404, but it doesn't mean that the user is not logged in anymore. Only when you receive 401 you should log out. And this should happen for every request, not only for "get user". Do not mix things up, think well in what situations each function can be used and take into consideration every possibility.